### PR TITLE
feat(docs): Add sample form/function code to the function docs

### PR DESCRIPTION
**Pull request by KyleAMathews**
Get people going quick if they want to try adding a form that submits to a function.

### CODE REVIEW

File: docs/docs/how-to/functions/getting-started.md [hunk 1]
```diff
@@ -123,4 +123,65 @@ export default async function postNewPersonHandler(req, res) {
...
+
+```js:title=src/api/form.js
+export default function formHandler(req, res) {
+  // req.body has the form values
+  console.log(req.body);
+
+  // Here is where you would validate the form values and
+  // do any other actions with it you need (e.g. save it somewhere or
+  // trigger an action for the user).
+  res.json(`OK`);
```
Comment (DSchau) [inline docs/docs/how-to/functions/getting-started.md:136]: Nit, but in the past I've tested these kinds of things (e.g. integration or unit test) and returning makes it easier to intercept and/or "mock" just to make sure things are wired up. I usually do something like:

Comment (KyleAMathews) [reply docs/docs/how-to/functions/getting-started.md]: 💯 — example code should be as perfect as possible (while not obscuring understanding the code for first time readers)

File: docs/docs/how-to/functions/getting-started.md [hunk 2]
```diff
@@ -123,4 +123,65 @@ export default async function postNewPersonHandler(req, res) {
 
 ## Forms
 
-Forms and Functions are often used together. See the [basic form example](https://github.com/gatsbyjs/gatsby-functions-beta/tree/main/examples/basic-form) as well as the [Forms doc page](/docs/how-to/adding-common-features/adding-forms/) which walks you through building a custom form.
+Forms and Functions are often used together. For a working example you can play with locally, see the [form example](https://github.com/gatsbyjs/gatsby-functions-beta/tree/main/examples/basic-form). The [Forms doc page](/docs/how-to/adding-common-features/adding-forms/) is a gentle introduction for building forms in React. Below is sample code for a very simple form that submits to a function that you can use as a basis for building out forms in Gatsby.
+
+```js:title=src/api/form.js
+export default function formHandler(req, res) {
+  // req.body has the form values
+  console.log(req.body);
```
Comment (DSchau) [inline docs/docs/how-to/functions/getting-started.md:131]: Could show a super basic example of validation, e.g.

```
if (!req.body.name) {
  return res.status(422).json({
    message: 'Name field is required'
  })
}
```

Comment (KyleAMathews) [reply docs/docs/how-to/functions/getting-started.md:131]: 👍

**Review by DSchau — APPROVED** [review]
👍

**Pull request merged**